draft-ietf-tls-cross-sni-resumption.md

```diff
@@ -95,18 +105,12 @@ The Flag
 
 Resumption across server names is negotiated using the TLS flags extension
 {{!I-D.draft-ietf-tls-tlsflags}}.  The server MAY send a
-resumption_across_names(8) flag in a NewSessionTicket message.  If the flag is
-sent, it indicates that the client MAY use the ticket for any SNI value for
-which the certificate presented by the server is valid.  The server MUST handle
-the ticket correctly by either resuming and using a new SNI provided by the
-client, or by ignoring the ticket.
-
-The server MAY send the flag if it reasonably believes that any server for any
-identity presented in its certificate would be capable of accepting that
-ticket.  The server SHOULD NOT send the flag otherwise, since, if the client
-follows the single-use ticket policy recommended by [RFC8446], sending the
-ticket results in it being no longer usable regardless of whether resumption
-has succeeded.
+resumption_across_names(8) flag in a NewSessionTicket message; the flag is an
+assertion by the server that any server for any identity presented in its
+certificate would be capable of accepting that ticket.  A client receiving a
+ticket with this flag MAY attempt resumption for any server name listed in the
```

Comment:
```suggestion
ticket with this flag MAY attempt resumption for any server name corresponding to an identity in the
```

Reply:
(We speak about identities in the cert and then pivot to names, so this just tries to tie them together.)